Looks great, there's only a minor failure on flake8 checks:
```
21:59:35  ./bodhi/tests/server/test_validators.py:454:1: E302 expected 2 blank lines, found 1
21:59:35  class TestValidateOverrideNotes(BasePyTestCase):
21:59:35  ^
```

--- a/bodhi/tests/server/test_validators.py
+++ b/bodhi/tests/server/test_validators.py
@@ -451,6 +451,7 @@ class TestValidateExpirationDate(BasePyTestCase):
         ]
         assert request.errors.status == exceptions.HTTPBadRequest.code
 
+
 class TestValidateOverrideNotes(BasePyTestCase):
     """Test the validate_override_notes() function."""
 

--- a/news/4182.bug
+++ b/news/4182.bug
@@ -1,0 +1,1 @@
+Fixed an issue with validators that prevents inconsistent refusal of bodhi override with maximum duration

